### ISSUE TITLE
To CMYK

### ISSUE THREAD

**Issue by jwb009** [whoisjuan/craft-color-mixer#13]
### What are you trying to do?

It would be nice to add a `toCmyk` filter to the plugin. CMYK conversion appears to be supported in the Iris package as of October '23

### What's your proposed solution?

Add a new Twig filter for `toCmyk`

**Comment by engram-design** [comment]
Happy to add, but from what I can tell they haven't tagged CMYK support in a release. Their latest release is [3.1.1](https://github.com/ozdemirburak/iris/tree/3.1.1) with no CMYK support.

**Comment by jwb009** [comment]
> Happy to add, but from what I can tell they haven't tagged CMYK support in a release. Their latest release is [3.1.1](https://github.com/ozdemirburak/iris/tree/3.1.1) with no CMYK support.

Thanks for looking into it Josh! Unless I'm mistaken, it looks like CMYK was added, but never documented. See https://github.com/ozdemirburak/iris/pull/44

**Comment by engram-design** [comment]
Yep, I can see it in their [master branch](https://github.com/ozdemirburak/iris) (and documented), but it's not included in any of their [releases](https://github.com/ozdemirburak/iris/releases). I'd _rather_ not be dependent on their master branch, compared to a tagged release if we can help it.

**Comment by jwb009** [comment]
Coming back to this again...looks like they have added support in their latest [release](https://github.com/ozdemirburak/iris/releases/tag/4.1.0).

**Comment by engram-design** [comment]
Great! Updated for the next release. To get this early, run `composer require verbb/color-mixer:"dev-craft-5 as 4.0.1"`.

I've also added OKLCH support thanks to the Iris update as well.

**Comment by engram-design** [comment]
Updated in [4.1.0](https://github.com/verbb/color-mixer/releases/tag/4.1.0).
